src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +18,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
```

Comment:
It's good practice to handle exceptions that may occur during the execution of the CompletableFutures to prevent the application from terminating unexpectedly.